great_tables/_scss.py

```diff
@@ -138,7 +138,10 @@ def compile_scss(data: GTData, id: Optional[str], compress: bool = True) -> str:
     if has_id:
         compiled_css = re.sub(r"\.gt_", f"#{id} .gt_", compiled_css, 0, re.MULTILINE)
         compiled_css = re.sub(r"thead", f"#{id} thead", compiled_css, 0, re.MULTILINE)
-        compiled_css = re.sub(r"^p \{", f"#{id} p " + "{", compiled_css, 0, re.MULTILINE)
+        if compress:
+            compiled_css = re.sub(r"^ p \{", f" #{id} p " + "{", compiled_css, 0, re.MULTILINE)
```

Comment:
I wonder if this could be condensed into a single regex?

```python
 re.sub(r"^( p|p) \{", f" #{id} p {{", compiled_css, 0, re.MULTILINE)
```

The main difference is using `^( p|p)` to capture that the difference is whether people has a space or not.

Here's how I tested:

```python
f = lambda x, id="ABC":  re.sub(r"^( p|p) \{", f" #{id} p {{", x, 0, re.MULTILINE)

# substitutes
f(" p {abc")
f("p {abc")

# no substitute (good!)
f("group {abc")

Reply:
Yep, that is totally good. I'll make the change!